content/en/blog/istio-resiliency-patterns/index.md

```diff
@@ -0,0 +1,542 @@
+---
+title: "What are the patterns of resilience in Istio and how to guarantee?"
```

Comment:
the "and how to guarantee?" part of the title seems off. Can you remove it and have the title be: "What are the patterns of resilience in Istio?"

Reply:
Done